src/pages/purchase/buzz.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Container,
+  Stack,
+  Title,
+  Text,
+  Alert,
+  Group,
+  Grid,
+  Paper,
+  List,
+  Center,
+  Divider,
+  ListProps,
+} from '@mantine/core';
+import { useRouter } from 'next/router';
+import { CurrencyIcon } from '~/components/Currency/CurrencyIcon';
+import { Currency } from '@prisma/client';
+import { BUZZ_FEATURE_LIST } from '~/server/common/constants';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { z } from 'zod';
+import { BuzzPurchase } from '~/components/Buzz/BuzzPurchase';
+import { enterFall, jelloVerical } from '~/libs/animations';
+import { EdgeMedia } from '~/components/EdgeMedia/EdgeMedia';
+import React, { useState } from 'react';
+import { CurrencyBadge } from '~/components/Currency/CurrencyBadge';
+
+const schema = z.object({
+  returnUrl: z.string().optional(),
+  minBuzzAmount: z.coerce.number().optional(),
+});
+
+const BuzzFeatures = (props: Omit<ListProps, 'children'>) => {
+  return (
+    <List listStyleType="none" spacing="sm" {...props}>
+      {BUZZ_FEATURE_LIST.map((feature) => (
+        <List.Item key={feature}>
+          <Group noWrap>
+            <CurrencyIcon style={{ flexShrink: 0 }} currency={Currency.BUZZ} size={18} />
+            <Text>{feature}</Text>
+          </Group>
+        </List.Item>
+      ))}
+    </List>
+  );
+};
+export default function PurchaseBuzz() {
+  const router = useRouter();
+  const { returnUrl, minBuzzAmount } = schema.parse(router.query);
+  const [success, setSuccess] = useState<boolean>(false);
+
+  const handlePurchaseSuccess = () => {
+    if (returnUrl) {
+      window.open(returnUrl, '_blank');
+    }
+
+    setSuccess(true);
+  };
+
+  if (success) {
+    return (
+      <Container size="md" mb="lg">
+        <Center
+          sx={{
+            // animation: `${jelloVerical} 2s 1s ease-in-out`,
+            animationName: `${enterFall}, ${jelloVerical}`,
```

Comment:
Just a little nit, can we rename this?
```suggestion
            animationName: `${enterFall}, ${jelloVertical}`,
```